internal/testdata/models/pet.go

```diff
@@ -32,3 +34,100 @@ func (TopPetOwner) ViewDef(dialect string) []gormschema.ViewOption {
 	}
 	return []gormschema.ViewOption{gormschema.CreateStmt(stmt)}
 }
+
+type UserPetHistory struct {
+	UserID    uint `gorm:"primaryKey"`
+	PetID     uint `gorm:"primaryKey"`
+	CreatedAt time.Time
+}
+
+func (Pet) Triggers(dialect string) [][]gormschema.TriggerOption {
```

Comment:
```suggestion
func (Pet) Triggers() []gormschema.Trigger {
	return []gormschema.Trigger{
		gormschema.NewTrigger(gormschema.CreateStmts("RAW SQL")),
		gormschema.NewTrigger(gormschema.Name("TR_1")).
			AS("RAW SQL"),
	}
}
```

cc @a8m

Reply:
I'd use keep only the raw definitions at this stage. A fluent API can be added on top of it and can be translated to the raw definition, but I don't think it's needed for now.

Reply:
> gormschema.Trigger {

I like. That means we can change the ViewOptions also to be simply gormschema.View

Reply:
using `gormschema.View` makes a lot of sense; I'll add it to my backlog.